cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,22 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  url = '/cart.html';
+
+  clickAddToCart() {
+    cy.contains('Add to cart').click();
+    cy.on('window:alert', (text) => {
```

Comment:
This class extends a PageObject class which already has alert handling declared. Use it.